workflow/dumping-workflows.rst

```diff
@@ -25,10 +25,9 @@ Images of the workflow defined above::
 .. code-block:: terminal
 
     $ php dump-graph-dot.php | dot -Tpng -o dot_graph.png
-    $ php dump-graph-puml.php | java -jar plantuml.jar -p  > puml_graph.png
-
     # run this command if you prefer SVG images:
     # $ php dump-graph-dot.php | dot -Tsvg -o dot_graph.svg
+    $ php dump-graph-puml.php | java -jar plantuml.jar -p  > puml_graph.png
```

Comment:
this grouping is a bit confusing now, because it generates a `png` file, but follows a comment/description which is talking about `svg` 🤔

Reply:
wanted to "group" `dot` cli vs `java`
i propose a new way